lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -107,6 +107,10 @@ function dosomething_helpers_preprocess_partners_vars(&$vars) {
       if (!empty($image_node->field_image_landscape)) {
         $img = dosomething_image_get_themed_image($image_node->nid, 'landscape', '550x300');
         $vars['partner_info'][$delta]['image'] = $img;
+      } else {
```

Comment:
I think since the quick fix isn't needed, (we can just add a landscape crop to fix the error) we should probably take the time to fix this in a more sustainable way